cmd/common.go

```diff
@@ -15,6 +15,7 @@ type ValidateWriter struct {
 type CompileWriter struct {
 	appId  string
 	stream salv1.Compiler_CompileClient
+	batchSize int32
```

Comment:
This is not properly formatted, please run the sources through `gofmt` and maybe add that step to `Makefile` also.